SdkHelpers.ambuild

```diff
@@ -123,7 +126,7 @@ class SdkHelpers(object):
             # Requires new enough ambuild version for __file__ to be defined
             sdk_manifest_dir = os.path.join(os.path.dirname(__file__), 'manifests')
         except NameError:
-            sdk_manifest_dir = os.path.join(builder.sourcePath, 'hl2sdk-manifests', 'manifests')
+            sdk_manifest_dir = os.path.join(os.path.dirname(get_this_file_path()), 'manifests')
```

Comment:
Should we check for either one, rather than just one or the other? (To not break compat with anyone that may be relying on this).

Reply:
Previous fallback method was expecting you to have hl2sdk-manifests in a root directory of your project, which I don't think this code would break anyhow